airflow/www/templates/airflow/dags.html

```diff
@@ -249,10 +249,12 @@ <h2>DAGs</h2>
         }
         return false;
       }
-      all_dags = $("[id^=toggle]");
-      $.each(all_dags, function(i,v) {
+
+      var encoded_dag_ids = [];
+      $.each($("[id^=toggle]"), function(i,v) {
```

Comment:
`.dags input[id^=toggle][dag_id]` is a better selector I think.

Slightly simpler:

```js
$("#dags [id^=toggle-][dag_id]").map(function() {
    return encodeURIComponent($(this).attr('dag_id'));
}).get().join()
```

Reply:
Oh though given what we are doing elsewhere in the loop ignore all this.